docs/docs/resources/pipeline-components/kafka-source-connector.yaml

```diff
@@ -0,0 +1,43 @@
+# Kafka source connector
+- type: kafka-source-connector
+  name: kafka-source-connector # required
+  namespace: namespace # required, clean up jobs for the connector will run here
```

Comment:
I think it makes sense to have a kafka-connector file where resetter configs are set as well as error topic

Reply:
it's ok like this, it should be covered in the documentation about defaults.yaml as we discuessed